requirements/dev.txt

```diff
@@ -4,6 +4,7 @@ autopep8==1.2.4
 coverage==4.2
 coveralls==1.1
 django-autofixture==0.12.1
+django-extensions==1.7.8
```

Comment:
the packages looks ok to me, but I am still not clear whether we should use this or not.